api_server.py

```
@@ -228,19 +282,35 @@ def health():
 def root():
     try:
         now = datetime.now()
```

Ruff (0.15.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)